docroot/themes/custom/uids_base/scss/components/banner.scss

```diff
@@ -2,26 +2,54 @@
 @import "assets/scss/_utilities.scss";
 @import "components/banner/banner.scss";
 
-.banner--reversed *:not(.bttn) {
-  color: #fff;
+.banner.banner--gradient-left-reversed .bold-headline.bold-headline--negative {
+  text-shadow: none;
+  color: $secondary;
 }
 
-.banner__summary {
-  line-height: 1.7;
-
-  p {
-    font-weight: 300;
-    font-size: 1.3rem;
-    margin-top: 1.05rem;
-    margin-bottom: 1.875rem;
+.banner {
+  @include breakpoint(sm) {
+    min-height: 18rem;
   }
+}
 
-  strong {
-    font-weight: $font-weight-medium-bold;
-  }
+.bg-pattern--brain-reversed,
+.bg--gold {
+  .bttn--primary {
+    color: #fff;
+    background: $secondary;
+    border: 1px solid $secondary;
 
-  a {
-    font-weight: $font-weight-medium;
+    svg {
+      color: $primary;
+    }
   }
+}
```

Comment:
Are these general rules that could be moved into a non-banner related file?

Reply:
Yes, we should move these to the button component.